src/main/java/uk/gov/hmcts/darts/common/entity/NodeRegisterEntity.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = NodeRegisterEntity.TABLE_NAME)
+@Getter
+@Setter
+public class NodeRegisterEntity {
+
+    public static final String TABLE_NAME = "node_register";
+    public static final String NODE_GENERATOR = "nod_gen";
+    public static final String NODE_SEQUENCE = "nod_seq";
+    public static final String DEVICE_ID = "der_id";
+    public static final String COURTROOM_ID = "ctr_id";
+    public static final String NODE_ID = "node_id";
```

Comment:
I'm guessing this will become `nod_id` if the three letter acronym is nod like for the sequence?

Reply:
The node_id is an existing column but is now becoming the primary key. Are you suggesting the constant should be renamed or the column?

Reply:
This hasn't been renamed by DBA's, so I kept it the same